packages/server/controllers/index.js

```diff
@@ -16,3 +17,7 @@ export const app_controller = useLocal
 	: mongo_app_controller;
 
 export const user_controller = mongo_user_controller;
+
+export const browser_activity_controller = useLocal
+	? json_activity_controller
```

Comment:
`json_activity_controller` is supposed to handle only Desktop application data.

Either create `controllers/json/browserAcitivty.controller.js` (similar to `controllers/json/activity.controller.js`) and use it
or 
just simply use 
`export const browser_activity_controller =  mongo_browser_activity_controller`

Reply:
okey 👍